critiquebrainz/frontend/views/reports.py

```diff
@@ -14,7 +16,12 @@
 @login_required
 @admin_view
 def index():
-    results, count = SpamReport.list(limit=RESULTS_LIMIT)
+    results, count = db_spam_report.list_reports(limit=RESULTS_LIMIT, inc_archived=False)
+    if results:
+        for report in results:
+            report["user"] = db_users.get_by_id(report["user_id"])
+            report["review"] = db_revision.get_review(report["revision_id"])
+            report["review"]["user"] = db_users.get_by_id(report["review"]["user_id"])
```

Comment:
Can this part be improved so that we don't have to make a bunch of unnecessary SQL queries? One thing that can be done is inclusion of necessary user and review data in `list_reports` function if that data is required every time reports are listed. Can do that by JOINing and putting data in nested dictionaries nicely.

I think this is easier to do with raw SQL queries without having to spend a lot of time figuring out how to do that with SQLAlchemy.